Dockerfile.ubuntu

```
@@ -0,0 +1,37 @@
+FROM ubuntu:16.04
+
+#Base settings
+ENV HOME /root
+
+COPY requirements.txt /root/requirements.txt
+
+#Install ZeroNet
+RUN apt update \
+ && apt install -y python3-pip tor openssl \
+ gcc libffi-dev musl-dev make \
+ pkg-config \
+ && python3 -m pip install --user --upgrade pip
+
+RUN pip3 install -r /root/requirements.txt \
```

```suggestion
RUN pip3 -m venv ~/zeroenv \
&& source ~/zeroenv/bin/activate \
&& pip3 install -r /root/requirements.txt \
```
Or something similar, need a virtual environment and as I said tor never should be run as root. You only need to modify the torrc file, create and than enable the tor service. 

At reboot your Tor will not work. Even possible to create a new user just for zeronet and so the path ~/ will be very useful. 

Neither Tor not ZeroNet should be run as root. 
Virtual environment

---

I'm not sure if it's a good idea -- ZeroNet is the only service running in this docker after all, so extra nesting is needless.

---

I'm also not sure if this is really good idea, but I did that and it works.
So I can revert that and only run ZN and tor using non root user, or leave approach with virtuallenv.